Makefile.dep

```diff
@@ -86,8 +86,10 @@ endif
 # always select gpio (until explicit dependencies are sorted out)
 FEATURES_OPTIONAL += periph_gpio
 
-# always select power management if available
-FEATURES_OPTIONAL += periph_pm
+# always select power management if not building the bootloader
+ifneq (1, $(RIOTBOOT_BUILD))
+  FEATURES_OPTIONAL += periph_pm
+endif
```

Comment:
Does this save rom?

Reply:
Could you add a comment on why this is done?

Reply:
I expanded on the comment.